test/liquid_voting/delegations_test.exs

```diff
@@ -61,24 +64,36 @@ defmodule LiquidVoting.DelegationsTest do
       assert {:ok, %Delegation{} = delegation} = Delegations.create_delegation(args)
     end
 
-    test "create_delegation/1 with proposal url sets global boolean to false", context do
-      proposal_url = "https://www.someorg/proposalX"
+    test "create_delegation/1 with duplicate data returns error changeset", context do
+      Delegations.create_delegation(context[:valid_attrs])
+      assert {:error, %Ecto.Changeset{}} = Delegations.create_delegation(context[:valid_attrs])
+    end
 
-      args = Map.merge(context[:valid_attrs], %{proposal_url: proposal_url})
-      {:ok, %Delegation{} = delegation} = Delegations.create_delegation(args)
+    test "upsert_delegation/1 with duplicate delegator and proposal_url updates the respective delegation",
+         context do
+      args = Map.merge(context[:valid_attrs], %{proposal_url: @proposal_url})
+      {:ok, %Delegation{} = delegation1} = Delegations.create_delegation(args)
 
-      assert delegation.global == false
+      args = Map.merge(context[:update_attrs], %{proposal_url: @proposal_url})
+
+      assert {:ok, %Delegation{} = delegation2} = Delegations.upsert_delegation(args)
+      assert delegation1.organization_id == delegation2.organization_id
```

Comment:
```suggestion
      assert delegation_1.organization_id == delegation_2.organization_id
```
naming_suggestion: `delegation` and `modified_delegation` or something a little more descriptive than `1` and `2`?